### PR TITLE
feat(dashboard): added excel export option to dashboard

### PR DESCRIPTION
**Pull request by durini309**

### CODE REVIEW

File: src/app/core/services/files/excel.service.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   exportFile(locationsUpdates: any) {
+       // TODO: get values from language configuration file
+      const header = [ 'Usuario', 'Lugar', 'Dirección', 'Fecha', 'Actividad', 'Ubicación' ]
+      const fileData = []
+      locationsUpdates.forEach(update => {
+        fileData.push({
+           Usuario: update.user_fullname,
+           Lugar: update.place_name,
+           Dirección: update.place_address,
+           Fecha: moment(new Date(update.time.seconds * 1000)).format('DD MMM YYYY hh:mm a'),
```
Comment (cancinos) [inline src/app/core/services/files/excel.service.ts]: Use moment.js library instead

File: src/app/shared/components/card-map/card-map.component.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
-import { Timestamp } from '@firebase/firestore-types'
 import { Component, OnInit, Input, ViewChild, ElementRef, AfterViewInit, OnChanges } from '@angular/core'
-import { LastLocations } from './last-locations'
+import { ExcelService } from '../../../core/services/files/excel.service'
```
Comment (cancinos) [inline src/app/shared/components/card-map/card-map.component.ts]: Is there anyway to get this url shorten?

File: src/app/core/services/files/excel.service.ts (hunk 1 above)
Comment (durini309) [reply src/app/core/services/files/excel.service.ts]: I'm using moment there

File: src/app/shared/components/card-map/card-map.component.ts (hunk 2 above)
Comment (durini309) [reply src/app/shared/components/card-map/card-map.component.ts]: No without some research. Maybe we can make this better after the MVP

File: src/app/core/services/files/excel.service.ts (hunk 1 above)
Comment (cancinos) [reply src/app/core/services/files/excel.service.ts]: But then, why you need "new Date"?

**Pull request merged**